doc/source/Windows.rst

```diff
@@ -0,0 +1,106 @@
+Windows Tutorial 
+================
+
+This guide gives you a brief introduction to how volatility3 works and some demonstration on suite of plugins available from
+
+Acquiring memory
+----------------
+
+Volatility does not provide the ability to acquire memory. In this tutorial we will see how we can use  `WinPmem <https://github.com/Velocidex/WinPmem/releases/latest>`_ for this purpose.
+
+Listing Plugins
+---------------
+
+.. code-block:: shell-session
+
+    $ python3 vol.py --help | grep windows | head -n 5
+        windows.bigpools.BigPools
+        windows.cmdline.CmdLine
+        windows.crashinfo.Crashinfo
+        windows.dlllist.DllList
+                            Lists the loaded modules in a particular windows
+
+Using plugins
+-------------
+
+The following is the syntax to run volatility tool.
+
+.. code-block:: shell-session
+
+    $ python3 vol.py -f <path to memory image> plugin_name plugin_option
+
+
+Example
+-------
+
+Example 1
+~~~~~~~~~
+
+In this example we will be using memory dump from PragyanCTF'22. The dump is available `here <https://drive.google.com/file/d/1VPIdqIDFkOi3zGET2g00s1y-OeWOUqyi/view?usp=sharing>`_.
+We will limit the discussion to memory forensics with volatility3 and not extend to other parts of the challenges. 
+
+In windows memory forensics using volatility3, most of the times we do not require creating a ISF file. 
+
+.. code-block:: shell-session
+
+    $ python3 vol.py -f MemDump.DMP windows.pslist | head -n 10
+        
+        Volatility 3 Framework 2.0.2	PDB scanning finished                                
```

Comment:
It seems that the version of Volatility is different for each document. When explaining the examples, I think it would be better to unify the versions. Also, if it is a document that will be explained based on many people, it seems better to use the last release, version `2.0.1`.